### PR TITLE
Not require a PHP open tag for highlighting

### PR DESCRIPTION
**Pull request by takahashi-yugo**
see #13

### CODE REVIEW

**Comment by nikic** [comment]
Can you please also drop the existing `<?php` in the code blocks?

File: Book/conf.py [hunk 1]
```diff
@@ -18,6 +18,13 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #sys.path.insert(0, os.path.abspath('.'))
 
+# -- PHP code highlights configuration --------------------------------------------
+
+from sphinx.highlighting import lexers
```
Comment (lyrixx) [inline Book/conf.py]: In python (like in PHP) we use to group all import a the top of the file.

Comment (takahashi-yugo) [inline Book/conf.py]: thanks.
moved import statements to the top.

**Comment by nikic** [comment]
Merged as https://github.com/phpinternalsbook/PHP-Internals-Book/commit/26f215bf517ee77459e077bfc98b9e1a5b9ef819, thanks :) Works great, for example last example on http://www.phpinternalsbook.com/classes_objects/magic_interfaces_comparable.html.

**Pull request closed without merging**